src/main/java/emissary/pickup/PickUpPlace.java

```diff
@@ -180,6 +184,13 @@ protected void configurePickUpPlace() {
         }
 
         ALWAYS_COPY_METADATA_VALS = configG.findEntriesAsSet("ALWAYS_COPY_METADATA");
+
+        // Setup objectTraceLogger
+        useObjectTraceLogger = configG.findBooleanEntry("USE_OBJECT_TRACE_LOGGER", useObjectTraceLogger);
```

Comment:
My gut feeling is that we'll migrate away from enabling on a place-specific basis and will instead use a system-wide toggle such as a system property, environment variable, or startup parameter, with that toggle evaluated inside the ObjectTracing class itself.  Probably a bit more like the `Logger` model where we instrument log statements throughout the code and let the central configuration dictate whether that statement is ignored.